gcp/tests/test_connect_scheduling.py

```
@@ -0,0 +1,562 @@
+    def test_missing_shift_times_config_raises(self):
+        with patch.object(_fn_connect_module, "config") as mock_cfg:
+            mock_cfg.shift_times = None
+            with pytest.raises(RuntimeError, match="shift_times missing"):
+                get_followup_times("first_shift")
+
+
+class TestFormatTimeLabelConnect:
+    """format_time_label formats time objects as 12-hour strings."""
+
+    @pytest.mark.parametrize(
+        "t, expected",
```

Ruff (0.15.15)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)